source/sgp_mode/AnalysisTools.h

```diff
@@ -32,40 +32,73 @@
 
 // start of getNecessarySites methods
 
-/*
- *
- * Input: Takes in a cpu and the identifier for a specific task
+using Library = sgpl::OpLibrary<
```

Comment:
This is duplicated from code in GenomeLibrary.h and should be removed, remember to be careful to retain the correct versions of the code when doing merges